docs/explanation/oci-image.md

```diff
@@ -0,0 +1,26 @@
+# OCI Image
+
+### wordpress-image
+
+The wordpress-image is custom built to include a default set of plugins and themes. The list of
+plugins and themes can be found at the reference section of the
+[documentation](https://charmhub.io/wordpress-k8s/docs/reference?channel=edge). Since WordPress is
+an application running on php, required libraries and dependencies are installed during the build
+process.
+
+WordPress application installation is done at runtime during database connection setup. This can
+happen during database relation changed, database relation joined or database config changed
+events.
+To facilitate the WordPress installation process, WordPress cli is embedded in the OCI image during
+build step. The latest cli php archive file from source is used.
+
+Currently, WordPress version 5.9.3 is used alongside Ubuntu 20.04 base image. The Ubuntu base image
+is not being upgraded to 22.04 due to an unsupported php version 8 for
```

Comment:
This reads like we will never upgrade, perhaps reword it to say that it hasn't been upgraded yet due to this reason?